sh/tests/solutions/check-user.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
normally we are making the solution an executable too, even if it is not necessary since we use "bash", but since it was like this for the Go piscine we also kept this process.